docs/command_line_reference.rst

```diff
@@ -10,47 +10,97 @@ Simexpal instructions are written as:
 
 In the following, we list all simexpal instructions and their arguments.
 
-instances
----------
-Checks and eventually downloads the instances for the experiments.
+archive
+-------
+Archives all the experimental data into a ``data.tar.gz`` file within the same directory
+where the ``experiments.yml`` file is located.
+
+builds
+------
+Responsible to download Git repositories and install executables.
 It supports the following actions:
 
-:list: lists all the instances found in the experiments.yml file.
-   Available instances are shown in green, unavailable instances in red.
-:install: downloads all the missing instances if they are taken from a public repository.
-   With the argument ``--overwrite`` it will also download the available instances and
-   overwrite them.
-:process: caches information about instances.
-:run-transform: manually runs a transformation on instance files.
+:make: downloads a Git repository and executes all build commands.
+:purge: deletes all related build files. To confirm this action it needs the ``-f`` argument.
+:remake: rebuilds a build from scratch.
+
+All the above actions can be applied to a subset of builds according to the ``--revisions`` and
+positional ``builds`` argument. Not specifying an argument will select all respective elements, e.g.,
+not specifying the ``--revisions`` argument will lead to the selection of every revision.
+
+develop
+-------
+Responsible to download Git repositories and install executables. It also allows you to redo arbitrary
+build steps after changing local Git files to take over the local changes.
+
+The ``develop`` action can be applied to a subset of builds according to the ``--revisions`` and
+positional ``builds`` argument. Not specifying an argument will select all respective elements, e.g.,
+not specifying the ``--revisions`` argument will lead to the selection of every revision.
 
-experiment
-----------
+It further supports the following arguments, which each can be specified as an additional argument
+``--[argument]``:
+
+:``checkout``: deletes the local Git repository and (re-)clones it.
+:``compile``: (re-)compiles the build.
+:``configure``: (re-)configures the build.
+:``delete-source``: deletes the source directory when purging.
+:``install``: (re-)installs the build.
+:``purge``: deletes all related build files.
+:``recheckout``: deletes the local build Git repository, reclones, regenerates, reconfigures, recompiles
+    and reinstalls it.
+:``recompile``: recompiles and reinstall the build.
+:``reconfigure``: reconfigures, recompiles and reinstalls the build.
+:``regenerate``: (re-)regenerates the build.
+:``reinstall``: reinstalls the build.
+:``reregenerate``: regenerates, reconfigures, recompiles and reinstalls the build.
```

Comment:
Better specify these as `--checkout` etc.

Reply:
The same applies to other commands.

Reply:
Done.